kotlin-react-router-dom/src/main/kotlin/react/router/dom/helpers.kt

```diff
@@ -3,8 +3,8 @@ package react.router.dom
 import react.*
 import kotlin.reflect.KClass
 
-fun <P : RProps> withRouter(klazz: KClass<out Component<P, *>>): RClass<P> =
+fun <P : RProps> withRouter(klazz: KClass<out Component<P, *>>): ComponentClass<P> =
```

Comment:
I think we need some analysis for these changes. For example, it seems we should use `ComponentType` here because we should not know how `withRouter` HOC implemented. It can return `FunctionComponent` under the hood, but we are saying that it is Class. I recommend use `ComponentType` everywhere it is possible in libraries for returned types and use `ComponentClass` only if it is really needed.

Reply:
Looks like next PR (check required before)